book/plutocracy_report.ipynb

```diff
@@ -302,18 +912,75 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "One highly contentious proposal whose outcome does not change after filtering out whales was [this temperature check to choose which Eth <> BNB bridge to use for Uniswap v3 governance](https://snapshot.org/#/uniswap/proposal/0x6b8df360fdf73085b21fdf5eef9f85916fbde95621a3d454cb20fbe545ffc852). Even for the least popular choices, whales contribute the majority of votes. We also observe that whales held the overwhelming majority of voting power for the runner-up, \"LayerZero\", with 99.9% of voting power allocated to the proposal behind it. By comparison, \"Wormhole\" received 65.1% of voting power for this proposal coming from whales."
+    "One highly contentious proposal whose outcome did not change after filtering out whales was [this temperature check to choose which Eth <> BNB bridge to use for Uniswap v3 governance](https://snapshot.org/#/uniswap/proposal/0x6b8df360fdf73085b21fdf5eef9f85916fbde95621a3d454cb20fbe545ffc852). We see that even for the least popular choices, whales still contributed to the majority of the votes. We also observe that whales held ~37.7% of voting power for the runner-up, \"LayerZero\". By comparison, \"Wormhole\" received ~51.1% of voting power for this proposal coming from whales."
```

Comment:
The phrasing makes this sentence a little hard to understand. How's this:

"\"LayerZero\", the runner-up, received ~37.7% of the total whale voting power. By comparison, \"Wormhole\" received ~51.1%."

Reply:
This is better, will make changes to main.